.github/workflows/write-external-forks-comment.yml

```diff
@@ -0,0 +1,20 @@
+name: Write e2e tests for forks comment
+
+on:
+  workflow_run:
+    workflows: ["Pull Request Labeler"]
+    types:
+      - completed
+
+jobs:
+  comment:
+    if: ${{ github.event.pull_request.head.repo.fork }}
```

Comment:
Does this work? The pre-release comment has a much more convuluted method to get the PR number, and I suspect `github.event` will be related to the workflow_run event rather than the original PR